apis/v1alpha1/local_object_reference_types.go

```diff
@@ -53,22 +54,30 @@ type ServicesDefaultLocalObjectReference struct {
 	//   resource: foos
 	//   name: myfoo
 	//
-	// +optional
+	// +kubebuilder:validation:MinLength=1
```

Comment:
Remove min since the field is being defaulted.

Reply:
Updated, thanks!